packages/hint/docs/contributor-guide/how-to/test-rules.md

```diff
@@ -84,13 +84,13 @@ possible as it is less error-prone.
 const tests: HintTest[] = [
     {
         name: 'Name of the tests',
-        serverConfig: 'HTML to use',
         reports: [{
             message: 'Error message targeting the word "HTML"',
             position: { column: 0, line: 0 }
-        }]
-    },
-    { ... }
+        }],
+        serverConfig: 'HTML to use'
+    }
+    // ,{ ... }
```

Comment:
There are probably just as clear if we simplify to `// ...` like elsewhere. What do you think?